test/piper.test.js

```diff
@@ -14,6 +14,7 @@ describe('Piper', () => {
     tc.downloadTool
       .mockReturnValue('./piper')
     core.getInput
+      .mockReturnValueOnce('v1.10.0')
```

Comment:
Is this worth a dedicated test case?

Reply:
probably yes but I'm not very comfortable with jest yet. Won't have time to look into that today. feel free to push something if it is trivial for you, otherwise I will when I find time.

Reply:
added test cases